src/CloudNative.CloudEvents.Kafka/PartitionKeyAdapters/BinaryGuidPartitionKeyAdapter.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+namespace CloudNative.CloudEvents.Kafka.PartitionKeyAdapters
```

Comment:
Please use file-scoped namespaces for new code. (It reduces the indentation level.)

Additionally, please add a copyright notice to each new file (just copy it from existing files).

Reply:
Corrected as sugested